mobile/lib/modules/backup/providers/backup.provider.dart

```diff
@@ -217,8 +217,7 @@ class BackupNotifier extends StateNotifier<BackUpState> {
 
       final assetCountInAlbum = await album.assetCountAsync;
       if (assetCountInAlbum > 0) {
-        final assetList =
-            await album.getAssetListRange(start: 0, end: assetCountInAlbum);
+        final assetList = await album.getAssetListPaged(page: 0, size: 1);
 
         if (assetList.isNotEmpty) {
```

Comment:
If `assetList` could potentially be empty could the wrapping `assetCount > 0` be removed? Then it also shouldn't be necessary to query the asset count at all here.

Reply:
I don't think so as we do not want to list empty albums, that being said we can probably remove `if (assetList.isNotEmpty)`

What do you think @alextran1502 ?

Reply:
Honestly I haven't looked at the implementation of each, but it seems logical to me that an asset count greater than 0 would be synonymous with an non empty list of assets from 0 to asset count.

I don't think an album with 0 assets would be able to return a non zero amount of assets.

Reply:
Ok, i've uploaded another commit